build/build-sdk-images/tool/base/Dockerfile

```diff
@@ -30,12 +30,26 @@ RUN git clone -b $GRPC_RELEASE_TAG --depth=5 https://github.com/grpc/grpc /var/l
     cd /var/local/git/grpc && \
     git submodule update --init && \
     echo "--- installing protobuf ---" && \
-    cd third_party/protobuf && \
-    ./autogen.sh && ./configure --enable-shared && \
+    cd third_party/protobuf/cmake && \
+    mkdir -p build && cd build && \
+    cmake -Dprotobuf_BUILD_TESTS=OFF \
+          -DBUILD_SHARED_LIBS=ON \
+          -DCMAKE_BUILD_TYPE=Release \
+           .. && \
     make -j4 && make install && make clean && ldconfig && \
     export CFLAGS=-Wno-error && export CXXFLAGS=-Wno-error && \
+    echo "--- installing absl for grpc build stability ---" && \
+    cd /var/local/git/grpc/third_party/abseil-cpp && \
+    mkdir -p build && cd build && \
+    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE .. && \
+    make -j4 && make install && make clean && ldconfig && \
```

Comment:
Install absl because the log recommended when build
```
CMake Warning at third_party/abseil-cpp/CMakeLists.txt:74 (message):
  A future Abseil release will default ABSL_PROPAGATE_CXX_STD to ON for CMake
  3.8 and up.  We recommend enabling this option to ensure your project still
  builds correctly.
```